app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @post = Post.find(params[:id])
+    @comments = @post.comments
+  end
+
+  def new
+  end
+
+  def create
+  end
+
+  def edit
+  end
+
+  def update
+  end
+
+  def delete
+  end
+end
```

Comment:
You've had time to work on Rails a bit since submitting this assignment, but let me know if you have any questions about implementing CRUD functionality beyond index and show.